### ISSUE TITLE
Create SecDevOps diagram for cover

### ISSUE THREAD

**Issue by DinisCruz** [DinisCruz/Book_SecDevOps_Risk_Workflow#18]
A variation of this

![image](https://cloud.githubusercontent.com/assets/656739/19413548/3166ad9e-9328-11e6-9330-016198e1f0f5.png)

Not 100% about the titles of the 4 circles (maybe add JIRA workflow or secure code to it )

Image found at https://www.linkedin.com/pulse/devsecops-secdevops-difference-kumar-mba-msc-cissp-mbcs-citp

Book cover to be designed at https://www.canva.com/

**Comment by davevs** [comment]
I asked my Agile colleagues; they are quite fond of making nice pictures.
Examples: 
http://blog.xebia.com/wp-content/uploads/2013/08/scrum-process.jpg
http://blog.xebia.com/wp-content/uploads/2014/06/Dia11.png

**Comment by DinisCruz** [comment]
@davevs these are quite cool, what is the license they are released under? Can I use them in the book? (The book is CC licensed)

**Comment by davevs** [comment]
Most of our stuff is CC licensed; the second one I'm sure off. I'll ask for the first one.

**Comment by davevs** [comment]
Sorry, it's the other way around. You can see the CC logos in the left bottom corner.

**Comment by DinisCruz** [comment]
Humm, but even that 2nd one (http://blog.xebia.com/wp-content/uploads/2013/08/scrum-process.jpg) since it says 'non-commercial use' , does that include this book? (which although it is available for free and is CC licensed, there is a commercial angle to it  (i.e. readers can pay for it))

looking at https://creativecommons.org/faq/#does-my-use-violate-the-noncommercial-clause-of-the-licenses doesn't seem to clarify for this scenario

**Comment by davevs** [comment]
Honestly I don't know. Usually Xebia is all about 'sharing with the community'. I reached out to the creator of them, but he's on holiday this week. I suspect we don't consider 'donations' a commercial angle.

**Comment by DinisCruz** [comment]
thx, it will be good to confirm

**Comment by davevs** [comment]
I checked with our Company lawyer and as long as you mention the source and keep the book in the current 'pay if you want' construction Xebia will not consider this 'commercial use' and you are free to use the graphics :)

**Comment by DinisCruz** [comment]
very cool :)

and now we also documented this case and workflow

thx

**Comment by DinisCruz** [comment]
that is for both diagrams right?

**Comment by davevs** [comment]
Yes, and also for any other info graphic you find on blog.xebia.com (unless specifically stated otherwise).

I also asked our design studio to come up with an alternative for the 4 circle picture. He told me he already had an idea :)

**Comment by davevs** [comment]
Sent you an email with alternative cover picture; let's discuss further here.

**Comment by DinisCruz** [comment]
I like it

![image](https://cloud.githubusercontent.com/assets/656739/19819957/ec8cbc46-9d4e-11e6-9e93-58ffab8bcdb8.png)

I'm not fully sure about the 4 domains ("Software Engineering", "Quality Assurance", "Technology Operations" and "Security")

Let me try to see how it fits in the current cover design

**Comment by davevs** [comment]
Our artist just took the original and kept the text. Feel free to provide feedback.
